lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1213 @@
+  rootfs:
+    description:
+      - If true, the first argument refers to an exploded container on the file system.
+    type: bool
+  security_opt:
+    description:
+      - Security Options. For example security_opt "seccomp=unconfined"
+    type: str
+  shm_size:
+    description:
+      - Size of /dev/shm. The format is <number><unit>. number must be greater than 0.
+        Unit is optional and can be b (bytes), k (kilobytes), m(megabytes), or g (gigabytes).
+        If you omit the unit, the system uses bytes. If you omit the size entirely, the system uses 64m
+    type: str
+  sig_proxy:
+    description:
+      - Proxy signals sent to the podman run command to the container process.
+        SIGCHLD, SIGSTOP, and SIGKILL are not proxied. The default is true.
+    type: bool
+  stop_signal:
+    description:
+      - Signal to stop a container. Default is SIGTERM.
+    type: str
+  stop_timeout:
+    description:
+      - Timeout (in seconds) to stop a container. Default is 10.
+    type: int
+  subgidname:
+    description:
+      - Run the container in a new user namespace using the map with 'name' in the /etc/subgid file.
+    type: str
+  subuidname:
+    description:
+      - Run the container in a new user namespace using the map with 'name' in the /etc/subuid file.
+    type: str
+  sysctl:
+    description:
+      - Configure namespaced kernel parameters at runtime
+    type: str
+  systemd:
+    description:
+      - Run container in systemd mode. The default is true.
+    type: bool
+  tmpfs:
+    description:
+      - Create a tmpfs mount. For example tmpfs "/tmp:rw,size=787448k,mode=1777"
+    type: str
+  tty:
+    description:
+      - Allocate a pseudo-TTY. The default is false.
+    type: bool
+  uidmap:
+    description:
+      - Run the container in a new user namespace using the supplied mapping.
+    type: str
+  ulimit:
+    description:
+      - Ulimit options
+    type: str
+  user:
+    description:
+      - Sets the username or UID used and optionally the groupname or GID for the specified command.
+    type: str
+  userns:
+    description:
+      - Set the user namespace mode for the container.
+        It defaults to the PODMAN_USERNS environment variable.
+        An empty value means user namespaces are disabled.
+    type: str
+  uts:
+    description:
+      - Set the UTS mode for the container
+    type: str
+  volume:
+    description:
+      - Create a bind mount. If you specify, volume /HOST-DIR:/CONTAINER-DIR,
+        podman bind mounts /HOST-DIR in the host to /CONTAINER-DIR in the podman container.
+    type: list
+    aliases:
+      - volumes
+  volumes_from:
+    description:
+      - Mount volumes from the specified container(s).
+    type: list
+  workdir:
+    description:
+      - Working directory inside the container.
+        The default working directory for running binaries within a container is the root directory (/).
+    type: str
+"""
+
+EXAMPLES = """
+- name: Pull an image
```

Comment:
This is slightly confusing to me since there is no word of pulling in the task itself. I'd probably rename to

> Pull an image if it's not present locally and run a container

Reply:
oops, copypaste